tools/common.ps1

```diff
@@ -124,3 +124,37 @@ function Refresh-Path {
         [System.Environment]::GetEnvironmentVariable("Path","User")
     ) -match '.' -join ';'
 }
+
+function Collect-LiveKD {
+    param (
+        [Parameter()]
+        [string]$OutFile
+    )
+
+    $LiveKD = "C:\livekd64.exe"
```

Comment:
Just a thought: Will all these tools block us from running on arm64?

Reply:
Yeah, that's one reason I'm trying to move these to common.ps1, so we at least have one point of maintenance.

Reply:
Though prepare-machine should also be able to copy the arch-specific binaries from the CI repo onto the machine.